README.md

```diff
@@ -374,32 +374,34 @@ for await (let b of resp) {
 ```
 note: to get the date of response samples you could do this `console.log(new Date(resp['AAPL'][0].startEpochTime*1000))`
 
-#### Last trade
+#### Get the Latest trade
 
 ```ts
-lastTrade(
-  symbol: string)
-) => Promise<LastTradeObject>
+getLatestTrade(
+  symbol: string,
+  config: any
```

Comment:
`Config` is not required anymore, you could remove this from here and from the next example as well. (Adding it isn't break the code, but using the API without giving the `config` in every function is more user friendly in my opinion.)

Reply:
Then just put a `?` after the param to be exact.